test/phpunit/arSolrPlugin/lib/query/ArSolrExistsQueryTest.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+use PHPUnit\Framework\TestCase;
+
+require_once 'plugins/arSolrPlugin/lib/query/arSolrExistsQuery.class.php';
+
+/**
+ * @internal
+ *
+ * @covers \arSolrExistsQuery
+ */
+class ArSolrExistsQueryTest extends TestCase
+{
+    public function createSolrExistsQueryProvider(): array
+    {
+        return [
+            'New arSolrExistsQuery with blank field' => ['field' => ''],
+            'New arSolrExistsQuery with null field' => ['field' => null],
```

Comment:
@anvit and @melaniekung: thinking about this - should '' or null trigger a validation error in the constructor? What will happen if a query is executed against Solr with these values?

Reply:
hi steve, they both return parser errors